x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -215,6 +219,17 @@ export const AllCasesList = React.memo<AllCasesListProps>(
           currentUserProfile={currentUserProfile}
           filterOptions={filterOptions}
         />
+        <CasesTableUtilityBar
```

Comment:
Lift up the component from `CasesTable` to `AllCasesList` to avoid prop drilling.